src/components/TodoCheckbox/TodoCheckbox.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { TickIcon, CrossIcon, IconsContainer } from "./TodoCheckbox.styled";
+import { ChatIcon } from "./TodoCheckbox.styled";
+
+function TodoCheckbox({ checkStatus }: { checkStatus: string }) {
```

Comment:
should use string literal type instead of `string`

https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types